stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -17,7 +17,7 @@ def check_qty(self):
             if record.quants_lines:
                 move = self.env['stock.move'].browse(
                     self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
+                if record.lines_qty > move.product_qty:
```

Comment:
Why do you change this? The correct value is put in product_uom_qty, not product_qty. Does this changes in 9.0?